src/Eurofurence.App.Server.Services/Abstractions/MinIO/MinIOConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace Eurofurence.App.Server.Services.Abstractions.MinIO
+{
+    public class MinIoConfiguration
+    {
+        public string Endpoint { get; set; }
+        public string AccessKey { get; set; }
+        public string SecretKey { get; set; }
+        public bool Secure { get; set; }
+        public string Bucket { get; set; }
+
+        public static MinIoConfiguration FromConfiguration(IConfiguration configuration)
```

Comment:
Instead of using this method, you can just let the configuration library bind the class automatically.
Just call `services.Configure<MinIoConfiguration>(configuration.GetSection("MinIo"))`, then you can inject the configuration as either `IOptionsSnapshot<MinIoConfiguration>` for transient services or `IOptionsMonitor<MinIoConfiguration>` for singleton services

Reply:
I wanted to do it that way, but I noticed that all other configurations use a FromConfiguration method like this, so I did it the same way for now. I think that the reason is, that we are using Autofac so IOptions with services.Configure doesn't work for the CLI toolbox for example.
So I did it the same way it was done in the other configurations for now, but I plan to replace this by the modern way of using IOptions for all configurations at some point, when we have removed Autofac

Reply:
Sounds good!